src/PHPDocker/Template/README.md.twig

```diff
@@ -59,6 +59,12 @@ php-fpm|php-fpm|9000
         * Run symfony console, `docker-compose exec php-fpm bin/console`
         * Open a mysql shell, `docker-compose exec mysql mysql -uroot -pCHOSEN_ROOT_PASSWORD`
 
+# Application file permissions #
+
+As in all server environments, your application needs the correct file permissions to work proberly. You can change the files throught the container, so you won't care if the user exists or has the same ID on your host.
+
+`sudo docker exec -it <yourName>-php-fpm chown -R www-data:www-data /application/public`
```

Comment:
Thank you for your pr :+1: 

This should work with a simpler `docker-compose` command:
```
docker-compose exec php-fpm chown -R www-data:www-data /application/public
```

without the need for sudo. What's your host? mac, win or linux?

Reply:
I'm on Linux and because of security concerns I keep the need for sudo to start containers. Don't want to setup my Linux, so docker works without sudo.

The docker-compose should do the same

Reply:
I understand, but most people run docker locally in a manner that does not require sudo (eg by adding themselves to the `docker` group) and in fact in Mac using sudo with docker can have unintended consequences in file ownership.